src/UI/CsobControl.php

```diff
@@ -146,6 +151,33 @@ public function pay()
 
 
 	/**
+	 * @param Csob\Payment $payment
+	 * @return Csob\Message\Response
+	 * @see https://github.com/csob/paymentgateway/wiki/eAPI-v1.6-EN#return-values-7
+	 */
+	private function oneclickPayment(Csob\Payment $payment)
+	{
+		// Note: We recommend to call payment/status not earlier than 2-3 seconds after payment/oneclick/start call.
```

Comment:
This node should rather be in the PhpDoc, as it can be viewed using QuickDocumentationReference shortcut in the IDE, without jumping to the method.